### ISSUE TITLE
[akka-scala] Bug with escaped Scala reserved words

### ISSUE THREAD

**Issue by mrdziuban** [swagger-api/swagger-codegen#5333]
##### Description

Reserved Scala keywords are [escaped using backticks](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AkkaScalaClientCodegen.java#L160), but since jmustache commit [9eda17f](https://github.com/samskivert/jmustache/commit/9eda17f85bfd8849850bdafa92d2f6ffe12bb71f), backticks are escaped using `&#x60;`, resulting in invalid Scala code.

##### Swagger-codegen version

\>= 2.2.0

##### Swagger declaration file content or url

This YAML:

```yml
swagger: '2.0'
paths:
  '/test/{private}':
    get:
      operationId: testEscaping
      parameters:
        - name: private
          in: path
```

generates this Scala method:

```scala
def testEscaping(&#x60;private&#x60;: String): ApiRequest[Unit] =
  ApiRequest[Unit](ApiMethods.GET, "https://localhost", "/test/{private}", "application/json")
    .withPathParam("private", &#x60;private&#x60;)
```

##### Command line used for generation

```bash
java -jar swagger-codegen-cli.jar \
  -l akka-scala \
  -i test_escaping.yaml \
  -t modules/swagger-codegen/src/main/resources/akka-scala \
  -o /tmp/test_escaping
```

##### Steps to reproduce

Name a parameter using any of the [reserved words listed here](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AkkaScalaClientCodegen.java#L79-L83).

##### Suggest a Fix

Specifying the `--reserved-words-mappings` option works as a workaround, but maybe this should use a leading underscore by default instead of backticks.

**Comment by wing328** [comment]
@mrdziuban thanks for reporting the issue. I think another way to fix it is to use `{{{ ...}}}` (original value) instead of `{{...}}`.

FYI. We're trying to consolidate `scala`, `async-scala` and `akka-scala` generator into one in 2.3.0 branch: https://github.com/swagger-api/swagger-codegen/issues/3812

**Comment by wing328** [comment]
Fixed via #5339. Please pull the latest master to give it a try.
